nifi-commons/nifi-record-path/src/main/java/org/apache/nifi/record/path/filter/NotEqualsFilter.java

```diff
@@ -30,7 +30,7 @@ public NotEqualsFilter(final RecordPathSegment lhs, final RecordPathSegment rhs)
     protected boolean test(final FieldValue fieldValue, final Object rhsValue) {
         final Object lhsValue = fieldValue.getValue();
         if (lhsValue == null) {
-            return rhsValue != null;
+            return false;
```

Comment:
What's the reason for this change? Not saying it's wrong, just trying to understand it :)